editor/editor_file_system.cpp

```diff
@@ -2046,14 +2046,20 @@ Error EditorFileSystem::_reimport_group(const String &p_group_file, const Vector
 	return err;
 }
 
-Error EditorFileSystem::_reimport_file(const String &p_file, const HashMap<StringName, Variant> &p_custom_options, const String &p_custom_importer, Variant *p_generator_parameters) {
+Error EditorFileSystem::import_file(const String &p_file, bool p_update_file_system) {
```

Comment:
The second argument name does not match declaration.

Reply:
Done!